patch/kernel/archive/sunxi-5.15/0011-add-initial-support-for-orangepi3-lts.patch

```diff
@@ -0,0 +1,656 @@
+From d0fd451c120f42098aa2a44b3ca514772e1ca42b Mon Sep 17 00:00:00 2001
+From: afaulkner420 <afaulkner420@gmail.com>
+Date: Fri, 25 Mar 2022 20:43:32 +0000
+Subject: [PATCH 11/11] Add initial support for OrangePi3 LTS
+
+---
+ arch/arm64/boot/dts/allwinner/Makefile        |  41 +-
+ .../allwinner/sun50i-h6-orangepi-3-lts.dts    | 398 ++++++++++++++++++
+ arch/arm64/boot/dts/allwinner/sun50i-h6.dtsi  | 101 ++++-
+ 3 files changed, 487 insertions(+), 53 deletions(-)
+ create mode 100644 arch/arm64/boot/dts/allwinner/sun50i-h6-orangepi-3-lts.dts
+
+diff --git a/arch/arm64/boot/dts/allwinner/Makefile b/arch/arm64/boot/dts/allwinner/Makefile
+index a14ad689b..ebd68cef7 100644
+--- a/arch/arm64/boot/dts/allwinner/Makefile
++++ b/arch/arm64/boot/dts/allwinner/Makefile
+@@ -1,52 +1,13 @@
+ # SPDX-License-Identifier: GPL-2.0
+-dtb-$(CONFIG_ARCH_SUNXI) += sun50i-a64-amarula-relic.dtb
```

Comment:
Why those are getting removed? Kernel is common.

Reply:
This is an old patch. That should have been removed. I'll change that...